blog/_posts/2018-07-19-team-sunshine.md

```diff
@@ -0,0 +1,117 @@
+
+In Bable offical website, there is [Babel REPL](https://babeljs.io/repl/), a playground for ES6; where user can write ES6 code, check the relevant code in ES5; and also the output. 
+
+<center>
+<img src="http://2ality.com/2015/08/getting-started-es6/babel-repl.png"/>
+</center>
+
+Currently, the [Babel core team](https://babeljs.io/team) is going to replace CodeMirror to [Ives'](https://twitter.com/CompuIves) [CodeSandBox](https://codesandbox.io/), the new code online editor. An end goal is to show a different view of the output code: [AST](https://astexplorer.net/), output code, or [time travel](https://github.com/babel/babel-time-travel). We have just started to work for integration time travel in REPL and are going to contribute for UI development with help of our mentors and coaches.
+
+## How we work
+We are part-time sponsored team and we meet from 6pm to 10pm at [PeachTree](https://www.coworker.com/south-korea/seoul/peachtree), a co-working space that supports startups via its networks, spaces, amenities and various programs. The name 'PeachTree' comes from the tale of 'Romance of the Three Kingdoms'. Just like how Yubi, Gwanwoo and Jangbi pleged their brotherhood under the peach tree. We hope to cooperate in harmony each other in its space under the peach tree.
+
+We have regular meeting with all members. Our team has 6 different time zones; Seoul, Lisbon, NY and more! 
+
+<div>
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">☀️<a href="https://twitter.com/hashtag/RGSoC2018?src=hash&amp;ref_src=twsrc%5Etfw">#RGSoC2018</a> <a href="https://twitter.com/hashtag/RGSoCSunshine?src=hash&amp;ref_src=twsrc%5Etfw">#RGSoCSunshine</a> DAY8 <a href="https://twitter.com/MarinaGJCho?ref_src=twsrc%5Etfw">@MarinaGJCho</a> <a href="https://twitter.com/sujinleeme?ref_src=twsrc%5Etfw">@sujinleeme</a><br>✅ We had two 📞; with <a href="https://twitter.com/AnaSofiaPinho?ref_src=twsrc%5Etfw">@AnaSofiaPinho</a>, and  <a href="https://twitter.com/existentialism?ref_src=twsrc%5Etfw">@existentialism</a> <a href="https://twitter.com/left_pad?ref_src=twsrc%5Etfw">@left_pad</a> <a href="https://twitter.com/CompuIves?ref_src=twsrc%5Etfw">@CompuIves</a> <br>✅ Checked Sandpack PR to REPL <br>✅ Ready to review code of current babel-time-travel <br>👉 Read More? <a href="https://t.co/M7oEjz8ygA">https://t.co/M7oEjz8ygA</a> <a href="https://t.co/ZraU9wJzCR">pic.twitter.com/ZraU9wJzCR</a></p>&mdash; Sujin Lee (@sujinleeme) <a href="https://twitter.com/sujinleeme/status/1017263419802521600?ref_src=twsrc%5Etfw">July 12, 2018</a></blockquote>
+<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+</div>
+
+## Meet shining our supervisor, mentors, and coaches!
+
+<img src="/img/blog/2018/2018-07-19-team-sunshine-group-call" alt="team sunshine with supervisor, coaches and mentors photo"/>
```

Comment:
Along with the common comment for all images in the file, please check the file path and name for this image. It seems to be broken for some reason.

Reply:
Solved: 5d7288c